playback/service/src/main/java/de/danoeh/antennapod/playback/service/internal/PlaybackServiceNotificationBuilder.java

```diff
@@ -93,6 +93,7 @@ public void loadIcon() {
                 Log.e(TAG, "Error loading the media icon for the notification", tr);
             }
         } catch (InterruptedException ignore) {
+            Thread.currentThread().interrupt();
```

Comment:
Why this? This seems unrelated to chapter display

Reply:
Ah, nice find! I would say we should do this in a new PR (no issue needed in this case). Then it becomes its own commit. This makes sure that if in the future someone wonders why this was done, the commit message is more focused and explanatory on one thing instead of mixing different things.